test_scripts/Defects/5_0/2448_SDL_does_not_clear_app_info_after_MASTER_RESET/2448_2_SDL_clears_app_info_after_MASTER_RESET(after_ign_off).lua

```diff
@@ -0,0 +1,36 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@IGapchuk Please explain why do we need 2nd script to check the issue?
Please notice ATF creates `app_info.dat` file automatically on each SDL start.
If such creation is not required `isAppInfoUpdate` parameter can be set in `SDL:StartSDL()` function
However it may affect `actions.mobile.allowSDL()` function since `webengineUniqueId` will be unknown and script may fail for `EXT` flow